geopandas/__init__.py

```diff
@@ -26,3 +26,6 @@
 from . import _version
 
 __version__ = _version.get_versions()["version"]
+
+from . import _version
+__version__ = _version.get_versions()['version']
```

Comment:
@EwoutH are you aware if there's a chance of updating versioner without causing this duplication? We can obviously push a commit to clear this but would be better we don't have to.

Reply:
I think it’s a formatting thing. As you can see in the diff below, there are a lot of formatting changes. Notice the line breaks and single versus double quotes.

I think our versioneer files are (at some point) formatted with Black, making them different from the originals.

Reply:
We have a pre-commit hook that does the formatting. It seems that there will be needed a manual commit to make these updates clean.

Reply:
Yeah I agree that’s not perfect, but at least it gives an heads-up about a new version and does most of the work.